package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "build": "node utils/build.js",
     "start": "node utils/server.js",
+    "release": "node scripts/release.js",
```

Comment:
It seems that release.js just do some bumping version work, maybe we can change another name like bump-version.js?

Reply:
Sure thing.